package.json

```diff
@@ -67,8 +67,7 @@
   "lint-staged": {
     "packages/**/*.{ts,tsx}": [
       "eslint --fix",
-      "prettier --write",
-      "tsc --noEmit --composite false"
```

Comment:
Can you undo this change, use `--no-verify` flag in case you feel it takes lot of time